### PR TITLE
feature/popover

### PR DESCRIPTION
**Pull request by cwatsonc**
#### Checklist

* [x ] Use a separate branch in your local repo (not `master`).
* [x ] I've read and agree with the [contribution guidelines](https://www.freeboardgames.org/docs/?path=/story/documentation-getting-started-contributing--page).

New design for client with popover cribbage board fixing a number of design issues and normalizing the behaviors for deals.

### CODE REVIEW

**Comment by cwatsonc** [comment]
This PR has a change to storybook -- not sure if those are auto-merged or not. @vdfdev

File: web/src/games/cardtable/game.test.ts [hunk 1]
```diff
@@ -3,113 +3,113 @@ import { Local } from 'boardgame.io/multiplayer';
...
+//     client.moves.cutDeck(1);
+//     const { G } = client.store.getState();
+
+//     expect(G.deck[0]).toEqual({ id: cardEnum.SK, rank: 13, faced: false });
+//     expect(G.deck[1]).toEqual({ id: cardEnum.SJ, rank: 11, faced: false });
+//     expect(G.deck[2]).toEqual({ id: cardEnum.H9, rank: 48, faced: false });
+//     expect(G.deck[3]).toEqual({ id: cardEnum.H2, rank: 41, faced: false });
+//   });
+
+//   test('cutDeck(-3) should rotate deck by one card', () => {
```
Comment (vdfdev) [inline web/src/games/cardtable/game.test.ts:20]: Why are you commenting out all these tests? Are they broken? Can you fix them?

Comment (cwatsonc) [reply web/src/games/cardtable/game.test.ts:20]: They are emitting errors that I've mentioned on bgio and fbg discord.  The errors appear benign but I was trying to isolate and investigate their cause.  I'll add the tests back -- this was a miss.

**Review by vdfdev — APPROVED** [review]

File: web/src/games/cardtable/game.test.ts (hunk 1 above)
Comment (cwatsonc) [reply web/src/games/cardtable/game.test.ts]: added back the tests

**Pull request merged**